spec/requests/post_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe 'Post', type: :request do
+  describe 'GET /index' do
+    it 'returns http success for post index' do

```

Comment:
- [Optional] Dear @Mike47ip, Generally, when we talk about a describe block it is expected to have a message that explains what all the tests inside of each `it` block will be testing. I can see you have a message `GET /index` for the `Describe` block, but your test cases do more than that. A better `describe` message I would suggest is something like `test the post index and show routes are working as expected`